src/client/Microsoft.Identity.Client/ApiConfig/Parameters/AcquireTokenForClientParameters.cs

```diff
@@ -14,7 +14,9 @@ internal class AcquireTokenForClientParameters : IAcquireTokenParameters
 
         /// <summary>
         /// </summary>
-        public bool SendX5C { get; set; }
+        public bool SendX5C { get; set; } 
+        
+        internal bool SetPerRequestX5C = false;
```

Comment:
The amount of repetition here makes me wonder if there should be an IConfidentialClientRequestParameters.

Reply:
There is, it's called `AbstractClientAppBaseAcquireTokenParameterBuilder`